client/layout/activity-panel/wordpress-notices.js

```diff
@@ -202,4 +205,10 @@ class WordPressNotices extends Component {
 	}
 }
 
+WordPressNotices.propTypes = {
+	showNotices: PropTypes.bool,
+	togglePanel: PropTypes.func,
+	onCountUpdate: PropTypes.func,
```

Comment:
While unlikely the scenario will ever be hit, should we add a default prop for `onCountUpdate` to a `noop` just to be safe here?